### PR TITLE
Clipboard ddl

### PR DESCRIPTION
**Pull request by jzviaginSingular**

### CODE REVIEW

File: ios/SingularBridge.m [hunk 1]
```diff
@@ -60,6 +60,7 @@ +(void)startSessionWithUserActivity:(NSUserActivity*)userActivity{
 
     // Singular Links fields
     singularConfig.launchOptions = launchOptions;
+    singularConfig.clipboardAttribution = [singularConfigDict objectForKey:@"clipboardAttribution"];
```
Comment (tapasyat) [inline ios/SingularBridge.m:63]: duplicate code in line 63 and 81

File: SingularConfig.js [hunk 2]
```diff
@@ -33,13 +34,19 @@ export class SingularConfig {
...
         this.enableLogging = false;
+        this.clipboardAttribution = false;
     }
 
     withSessionTimeoutInSec(sessionTimeout) {
         this.sessionTimeout = sessionTimeout;
         return this;
     }
 
+    withClipboardAttribution() {
```
Comment (shefdanny) [inline SingularConfig.js:45]: could it be better to give the dev the option to pass a boolean value? if they have some logic on their side to decide? the current impl is limiting IMO

**Review by shefdanny — APPROVED** [review]
see my comment, up to you, not critical. otherwise approved

File: SingularConfig.js (hunk 2 above)
Comment (jzviaginSingular) [reply SingularConfig.js:45]: its false by default

**Pull request merged**